src/data/2025/dev.js

```diff
@@ -5,68 +5,189 @@ import Rehanashri from '@/data/2025/assets/images/dev/rehanashri.webp'
 import ShugmiShumunov from '@/data/2025/assets/images/dev/shugmi_shumunov.webp'
 import KayleeBusenbark from '@/data/2025/assets/images/dev/kaylee_busenbark.webp'
 import JustinBeaudry from '@/data/2025/assets/images/dev/justin_beaudry.webp'
+import HudaHussaini from '@/assets/images/facilitators/Huda_Hussaini.webp'
+import ZavaarShah from '@/assets/images/facilitators/Zavaar_Shah.webp'
+import JingranWang from '@/assets/images/facilitators/Jingran_Wang.webp'
+import JuanLinaresOrihuela from '@/assets/images/facilitators/Juan_Linares_Orihuela.webp'
+import SumiaSaleh from '@/assets/images/facilitators/Sumia_Saleh.webp'
+import DaveKoziel from '@/assets/images/organizers/Dave_Koziol.png'
+import JennaRitten from '@/assets/images/organizers/Jenna_Ritten.png'
 
 export const devTeamData = [
   {
+    id: 1,
+    name: 'Dave Koziel',
+    twitter: 'davekoziol',
+    linkedin: 'https://www.linkedin.com/in/davekoziol/',
+    github: '',
+    devfest: 'organizer',
+    university: '',
+    organization: 'Little Caesars',
+    role: 'GDG Organizer',
```

Comment:
Don't mind the change from position to role - we'll have to be mindful to update the data accordingly where it's used